### PR TITLE
Add FAQ about add-on creation needing personal information

### PR DESCRIPTION
**Pull request by Kirbeerus**
## Describe your PR

Somme add-on need you to fill you account information to be able to create them.
Added a part in the FAQ about this.

## Checklist

- [x] My PR is related to an opened issue : #433 
- [x] I've read the [contributing guidelines](/CleverCloud/documentation/blob/main/CONTRIBUTING.md)

## Reviewers
_Who should review these changes?_ @

### CODE REVIEW

File: content/doc/find-help/faq.md [hunk 1]
```diff
@@ -210,3 +210,8 @@ If a VACUUM operation needs more disk that there is remaining, migrating to the
 Clever Cloud store all your backups on [Cellar](https://www.clever-cloud.com/product/cellar-object-storage/). 
 Cellar has a replication system to protect the data stored inside. It creates three copies of your backups, each stored in a different datacenters in the PAR region.  
 This way, even if one datacenter has an accident, your backup are still safe in two different datacenters.
+
+## I can't create my add-on
+
+The creation of certain add-ons require you to fill the information of your account, such as the city, ZIP code, ect..
```
Comment (cnivolle) [inline content/doc/find-help/faq.md:216]:
```suggestion
The creation of add-ons require you to fill the information of your account, such as the city, ZIP code, etc.
```

**Review by cnivolle — CHANGES_REQUESTED** [review]

**Comment by Kirbeerus** [comment]
@cnivolle Can you review the change you requested ? Otherwise, I can't launch the merge.

File: content/doc/find-help/faq.md [hunk 2]
```diff
@@ -215,3 +215,8 @@ If a VACUUM operation needs more disk that there is remaining, migrating to the
 Clever Cloud store all your backups on [Cellar](https://www.clever-cloud.com/product/cellar-object-storage/). 
 Cellar has a replication system to protect the data stored inside. It creates three copies of your backups, each stored in a different data centers in the PAR region.  
 This way, even if one data center has an accident, your backup are still safe in two different data centers.
+
+## I can't create my add-on
+
+The creation of add-ons require you to fill the information of your account, such as the city, ZIP code, etc.
```
Comment (cnivolle) [inline content/doc/find-help/faq.md:221]:
```suggestion
To create add-ons, you need to complete your account information, including your city and ZIP code. 
```

File: content/doc/find-help/faq.md [hunk 3]
```diff
@@ -215,3 +215,8 @@ If a VACUUM operation needs more disk that there is remaining, migrating to the
...
+For example, you can't create a Matomo add-on if those information aren't present.
```
Comment (cnivolle) [inline content/doc/find-help/faq.md:222]:
```suggestion
For instance, you cannot create a Matomo add-on until you provide these details.```

**Review by cnivolle — APPROVED** [review]

**Pull request merged**